Update _index.md (#388)
hyprwm/hyprland-wiki: pages/FAQ/_index.md

will pop into your `~/Pictures/` (You can configure grim and slurp, see their
GitHub pages).

> If you're wanting those screenshots to go directly to your clipboard, consider using `wl-copy`, from [`wl-clipboard`](https://github.com/bugaevc/wl-clipboard) or even `xclip`, an example binding may look something like this:
> `bind = SUPER_CTRL, S, exec, grim -g "$(slurp -d)" - | wl-copy`
> 

Reviewer: ```suggestion
If you want those screenshots to go directly to your clipboard, consider using `wl-copy`, from [`wl-clipboard`](https://github.com/bugaevc/wl-clipboard). Here's an example binding:
`bind = , Print, exec, grim -g "$(slurp -d)" - | wl-copy`
```

`xclip` is an X11 utility, not sure why you'd recommend it.

Author: fair
